src/Components/SiteColorsPicker.tsx

```diff
@@ -0,0 +1,70 @@
+import Color from 'color'
+import { ColorPicker, useColor } from 'react-color-palette'
+import { connect } from 'scrivito'
+import { HomepageInstance } from '../Objs/Homepage/HomepageObjClass'
+import './SiteColorsPicker.scss'
+
+export const SiteColorsPicker = connect(function SiteColorsPicker({
+  page,
+}: {
+  page: HomepageInstance
+}) {
+  const initialSiteColorPrimary = page.get('siteColorPrimary')
+  const [primaryColor, setPrimaryColor] = useColor(
+    initialSiteColorPrimary || '#274486',
+  )
+
+  const initialSiteColorSecondary = page.get('siteColorSecondary')
+  const [secondaryColor, setSecondaryColor] = useColor(
+    initialSiteColorSecondary || '#39a9eb',
+  )
+
+  return (
+    <div className="site-colors-picker">
+      <div>
+        <h2>Primary color</h2>
+        <ColorPicker
+          color={primaryColor}
+          hideInput={['hsv']}
+          hideAlpha={true}
+          onChange={(iColor) => {
+            const siteColorPrimary = iColor.hex
+            page.update({
+              siteColorPrimary,
+              siteColorPrimaryDarken: darken(siteColorPrimary),
+              siteColorPrimaryLighten: lighten(siteColorPrimary),
+            })
+
+            setPrimaryColor(iColor)
+          }}
+        />
+      </div>
+      <div>
+        <h2>Secondary color</h2>
+        <ColorPicker
+          color={secondaryColor}
+          hideInput={['hsv']}
+          hideAlpha={true}
+          onChange={(iColor) => {
+            const siteColorSecondary = iColor.hex
+            page.update({
+              siteColorSecondary,
+              siteColorSecondaryDarken: darken(siteColorSecondary),
+              siteColorSecondaryLighten: lighten(siteColorSecondary),
+            })
+
+            setSecondaryColor(iColor)
+          }}
+        />
+      </div>
+    </div>
+  )
+})
+
+function lighten(color: string): string {
+  return Color(color).lighten(0.5).hex()
+}
+
+function darken(color: string): string {
+  return Color(color).darken(0.5).hex()
+}
```

Comment:
If there is a widely supported css version of it, I'm fine with using it. But the question is: Is `hsl(from var(--color-primary) h s calc(l - 5))` well supported?

Reply:
My research so far did not show, that `hsl(from var(--color-primary) h s calc(l - 5))` is well supported.

Reply:
Looks like Firefox is late to the party https://caniuse.com/?search=relative%20color.
As mentioned on SO a more compatible approach is `color-mix`.

Screenshot from FF 125:
<img width="466" alt="image" src="https://github.com/Scrivito/scrivito-portal-app/assets/986170/b1a86839-6272-46eb-84b0-a056da8f9e51">

Reply:
However, we could postpone this as an optimization for a follow-up or some time later, since we would just remove attributes which is backwards compatible.

Reply:
https://caniuse.com/?search=relative%20color shows, that `color-mix` is also not supported on Safari 16.3 or earlier. Safari 16.4 was release about a year ago. So to me this seems to be not widely supported enough.

Reply:
True: Global	89.46%
FMI: What is our threshold?

Reply:
> FMI: What is our threshold?

We don't have a clearly defined threshold yet.

Reply:
Follow up: https://github.com/Scrivito/scrivito-portal-app/pull/711